go.work.sum

```diff
@@ -1,172 +1,2037 @@
+4d63.com/gochecknoglobals v0.1.0/go.mod h1:wfdC5ZjKSPr7CybKEcgJhUOgeAQW1+7WcyK8OvUilfo=
```

Comment:
Are all these changes needed?

Reply:
@p0mvn I have no idea why this is showing as a diff, its exactly the same as what is on main